### DISCUSSION TITLE
Improve repository/organization overview pages on Mend developer portal

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#30273] [Suggest an Idea]
### Tell us more.

PR #30132 adds this screenshot of the Mend developer portal to the docs:

![mend developer portal, settings button highlighted](https://github.com/user-attachments/assets/dc07ab31-8843-4eaa-a07c-1978731d0f2a)

The text/styling for the repository and organization page is confusing. 

I recommend you:

- [ ] Show a logo matching the page content
- [ ] Update the button text, so each page has a unique button text
- [ ] Add a heading that summarizes the page

See the table for more details:

| Which page? | Logo | Button text | Heading |
| --- | --- | --- | --- |
| Organization | Organization avatar | `Organization settings` | `$organization` overview |
| Repository | Generic repository logo | `Repository settings` | `$repository` overview |

**Comment by rarkins** [comment]
I disagree 

1. Repos do not have logos, only orgs do
2. Even on github.com, Settings is Settings: 
<img width="839" alt="image" src="https://github.com/user-attachments/assets/376667b2-e465-4268-af76-318dce816b95">

**Reply by HonkingGoose** [reply]
1. Showing the logo for organization accounts seems nice to have then?
2. Yes, but: GitHub is really clear where you are, and shows the organization or username logo as extra confirmation. Then when you use the Settings button, you get a page with sections on the left, and buttons to use in the middle. Each section and button also has good text explaining what it does.

The key thing is that the current text/layout is confusing me. I'll let you decide if/how you want to improve things.

**Reply by jamietanna** [reply]
Re 1 - we now do show the Organization/User profile photo when viewing it - https://developer.mend.io/github/justo-mend
